scripts/run.py

```diff
@@ -14,24 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Optional
 
-from .transformations_config import parse_transformation_configs
-from .utils import CDFToolConfig
+# from .transformations_config import parse_transformation_configs
```

Comment:
And delete here. Do we want to run transformations at all? We should probably rely on schedules found in the config. Maybe delete the entire file?